pv_site_api/fake.py

```diff
@@ -19,6 +24,28 @@
 fake_client_uuid = "c97f68cd-50e0-49bb-a850-108d4a9f7b7e"
 
 
+def make_fake_inverters() -> Inverters:
+    """Make fake inverters"""
+    inverter = InverterValues(
+        id="0",
+        vendor="Test",
+        chargingLocationId="0",
+        lastSeen="never",
+        isReachable="false",
+        productionState=InverterProductionState(
+            productionRate=0.1, isProducing=False, totalLifetimeProduction=0.5, lastUpdated="never"
+        ),
+        information=InverterInformation(
+            id="0", brand="tesla", model="x", siteName="ocf", installationDate="1-23-4567"
+        ),
+        location=InverterLocation(latitude=0.0, longitude=0.1),
+    )
+    inverters_list = Inverters(
+        inverters=[inverter],
+    )
```

Comment:
very tiny detail, but could we put in "better" fake data? the data displayed [here](https://developers.enode.com/api/reference#getInverter) might be suitable 

here it is in json form: 
```
{
  "id": "string",
  "vendor": "EMA",
  "chargingLocationId": "8d90101b-3f2f-462a-bbb4-1ed320d33bbe",
  "lastSeen": "2020-04-07T17:04:26Z",
  "isReachable": true,
  "productionState": {
    "productionRate": 0,
    "isProducing": true,
    "totalLifetimeProduction": 100152.56,
    "lastUpdated": "2020-04-07T17:04:26Z"
  },
  "information": {
    "id": "string",
    "brand": "EMA",
    "model": "Sunny Boy",
    "siteName": "Sunny Plant",
    "installationDate": "2020-04-07T17:04:26Z"
  },
  "location": {
    "longitude": 10.7197486,
    "latitude": 59.9173985
  }
}
```